RecoLocalCalo/HGCalRecAlgos/interface/HGCalRecHitSimpleAlgo.h

```diff
@@ -18,72 +18,67 @@
 #include <iostream>
 
 class HGCalRecHitSimpleAlgo : public HGCalRecHitAbsAlgo {
- public:
+public:
   // default ctor
   HGCalRecHitSimpleAlgo() {
     adcToGeVConstant_ = -1;
     adcToGeVConstantIsSet_ = false;
   }
 
-  void setLayerWeights(const std::vector<float>& weights) override {
-    weights_ = weights;
-  }
-  
+  void setLayerWeights(const std::vector<float>& weights) override { weights_ = weights; }
+
   void setADCToGeVConstant(const float value) override {
     adcToGeVConstant_ = value;
     adcToGeVConstantIsSet_ = true;
   }
-  
-  
+
   // destructor
-  ~HGCalRecHitSimpleAlgo() override { };
-  
+  ~HGCalRecHitSimpleAlgo() override{};
+
   /// Compute parameters
-  HGCRecHit makeRecHit(const HGCUncalibratedRecHit& uncalibRH,
-                               const uint32_t& flags = 0) const override {
-    
-    if(!adcToGeVConstantIsSet_) {
-      throw cms::Exception("HGCalRecHitSimpleAlgoBadConfig") 
-        << "makeRecHit: adcToGeVConstant_ not set before calling this method!";
+  HGCRecHit makeRecHit(const HGCUncalibratedRecHit& uncalibRH, const uint32_t& flags = 0) const override {
+    if (!adcToGeVConstantIsSet_) {
+      throw cms::Exception("HGCalRecHitSimpleAlgoBadConfig")
+          << "makeRecHit: adcToGeVConstant_ not set before calling this method!";
     }
-    
+
     DetId baseid = uncalibRH.id();
     unsigned layer = 0;
-    if( DetId::Hcal == baseid.det() && HcalEndcap == baseid.subdetId() ) {
-      layer =  HcalDetId(baseid).depth() + 40;
-    } else if ( DetId::Forward == baseid.det() && HGCEE == baseid.subdetId() ) {
+    if (DetId::Hcal == baseid.det() && HcalEndcap == baseid.subdetId()) {
+      layer = HcalDetId(baseid).depth() + 40;
```

Comment:
these hardcoded values should come from the geometry or RecHitTools

Reply:
Ciao @kpedro88,
as I said in the comment to this PR the package `RecoLocalCalo/HGCCalRecAlgos` has been added only to be able to include all `code-format` changes that were in #26991 
As such, I'd not consider this package as part of this review.

Reply:
Okay, I've added this to #25472.